rpmlint/checks/PolkitCheck.py

```diff
@@ -0,0 +1,95 @@
+import os
+from re import split
+from xml.dom.minidom import parse
+from xml.parsers.expat import ExpatError
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class PolkitCheck(AbstractCheck):
+    """Note: This expects the package polkit-default-privs to be installed to work."""
```

Comment:
I still would like to see more elaborate documentation about the purpose of this check:

```
This check enforces a whitelisting for polkit policies installed by packages. openSUSE uses the package
`polkit-default-privs` to apply one of a selection of default system profiles for polkit actions. A package's polkit
action is considered whitelisted if an entry for it is existing in polkit-default-priv's standard profile.

Apart from this this check also performs minor consistency checks and gives suggestions to packages regarding 
installed polkit policies.
```